staging/src/k8s.io/kubelet/pkg/apis/dra/v1alpha3/api.proto

```diff
@@ -45,11 +45,9 @@ service Node {
   rpc NodeUnprepareResources (NodeUnprepareResourcesRequest)
     returns (NodeUnprepareResourcesResponse) {}
 
-  // NodeListAndWatchResources returns a stream of NodeResourcesResponse objects.
-  // At the start and whenever resource availability changes, the
-  // plugin must send one such object with all information to the Kubelet.
-  rpc NodeListAndWatchResources(NodeListAndWatchResourcesRequest)
-    returns (stream NodeListAndWatchResourcesResponse) {}
+  // TODO: removing NodeListAndWatchResources and the code for
+  // publishing ResourceSlice objects by kubelet is an API break.
+  // If we do this, then v1alpha3 must be renamed to v1alpha4.
```

Comment:
renamed,. meaning `v1alpha3` is gone from the tree as `v1alpha4` is added?

Reply:
I can remove this TODO. The commit which renames v1alpha3 to v1alpha4 resolves it.

Reply:
Done: https://github.com/kubernetes/kubernetes/compare/bee827b1b42e53895b73ee6febb30ddf611797a9..36b913f2ed61a881e764e592f5e7b18984fca640